docs/ops/convolution/BinaryConvolution_1.md

```diff
@@ -4,20 +4,61 @@
 
 **Category**: *Convolution*
 
-**Short description**: *BinaryConvolution* convolution with binary weights, binary input and integer output
+**Short description**: Computes 2D convolution of binary input and binary kernel tensors.
+
+**Detailed description**: The operation bahaves as regular *Convolution* but uses specialized algorithm for computations on binary data. More thorough explanation can be found in [Understanding Binary Neural Networks](https://sushscience.wordpress.com/2017/10/01/understanding-binary-neural-networks/) and [Bitwise Neural Networks](https://saige.sice.indiana.edu/wp-content/uploads/icml2015_mkim.pdf).  
+
+
+Computation algorithm for mode *xnor-popcount*: 
+- X = XNOR(input_patch, filter) // bitwise XNOR operation on two bit streams
```

Comment:
Maybe use comma after the formula? Same as in line 15

Reply:
Done.